test/test_helper.rb

```diff
@@ -4,6 +4,7 @@
 require 'minitest/emoji'
 require 'capybara/rails'
 require 'pry'
+require 'mocha/mini_test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.